spec/rmagick/image_list/montage_spec.rb

```diff
@@ -94,4 +100,20 @@
       expect(ilist).to eq(@ilist)
     end.to raise_error(NoMethodError)
   end
+
+  it 'montages the image' do
+    imagelist = Magick::ImageList.new(IMAGES_DIR + '/Flower_Hat.jpg')
+
+    new_imagelist = imagelist.montage do
+      self.border_width = 100
+      self.border_color = 'red'
+      self.background_color = 'blue'
+      self.matte_color = 'yellow'
+      self.frame = '10x10'
+      self.gravity = Magick::CenterGravity
+    end
+
+    # montage ../../doc/ex/images/Flower_Hat.jpg -border 100x -bordercolor red -mattecolor yellow -background blue -frame 10x10 -gravity Center expected/montage_border_color.jpg
```

Comment:
Do we need this comment?

Reply:
@dlemstra we don't, but I think I'd like to keep it around for now if you don't mind. It gives me an idea for a matcher that might be useful:

```rb
expect(image).to match_im_command(['-border 100x', '-bordercolor red', ...])
```